packages/story-editor/src/components/footer/primaryMenu.js

```diff
@@ -50,6 +51,7 @@ function PrimaryMenu() {
       <MenuItems id="primary-menu-items">
         <ZoomSelector />
         <GridViewButton />
+        <Layers />
```

Comment:
If you have metaboxes (from other plugins), the alignment is a bit off (between layers button and layers panel).

IMO the layers panel button should be last (right-most) and the metaboxes button should be moved a bit, but UX might have a better suggestion.

![Screen Shot 2022-03-08 at 15 37 31](https://user-images.githubusercontent.com/637548/157320937-5225b72e-d188-4691-b3c7-9b0991299c30.png)

Reply:
I've noticed that panels on the left side of the footer are not always aligned to the button:

Checklist:
![image](https://user-images.githubusercontent.com/21036224/157439038-dd15167c-2e81-4a65-9e19-a8ac6789215c.png)

Shortcuts:
![image](https://user-images.githubusercontent.com/21036224/157439074-b2c25ee1-f16d-471f-9f47-a9b30028029d.png)

Not sure why - maybe to save the canvas overflow? that might be especially useful for the Layers panel that will be visible more often than the Shortcuts (so we want to cover as little of the canvas as possible).
I will ask Design pod about that.

Reply:
is this in figma?

Reply:
I will talk to Amy about alignment.

Reply:
Here is the Slack discussion: https://xwp.slack.com/archives/C01ESBB8ZN1/p1646935295628609?thread_ts=1646930940.483089&cid=C01ESBB8ZN1

Reply:
Talked to amy about what I have for this and she's fine with it as is. Alignment is either to the left or the right, not the center ever. Looks good with and without meta buttons - I added screenshots to the description.